Makefile

```diff
@@ -31,7 +33,21 @@ fmtcheck: ## Check go formatting
 	@gofmt -l $(SOURCES) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi
 
 test: ## Run unit tests
-	@go test ./pkg/...
+	@go test -cover ./pkg/...
+
+test-coverage-html: ## checkout the coverage locally of your tests
+	@echo "mode: count" > coverage-all.out
+	@$(foreach pkg,$(PACKAGES),\
+		go test -coverprofile=coverage.out -covermode=count $(pkg);\
+		tail -n +2 coverage.out >> coverage-all.out;)
+	@go tool cover -html=coverage-all.out # -o coverage.html
```

Comment:
So if you are excluding `-o coverage.html` where is the file that I would open in my browser to see it ?

Reply:
running `go tool cover -html=coverage-all.out` will actually open the browser window for you. With the `-o` it will simply write it to the file. I'll remove the reference to the coverage.html